### PR TITLE
Emojifyme signal enhancement for eaburns and me

### LINKED ISSUE #95: Don't reply with my previous message with emoji replaced
This is never funny and it's so annoying that I want to mute catbase.

### PR DESCRIPTION
**Pull request by mccoyst**
Fixes #95. Probably.

### CODE REVIEW

**Comment by skiesel** [comment]
I dislike the whole idea of making catbase less annoying.

BUT, if you must. Please add this into the GIANT config file so that we can easily add new tokens to ignore rather than hardcoding them.

File: plugins/emojifyme/emojifyme.go [hunk 1]
```diff
@@ -71,19 +71,25 @@ func (p *EmojifyMePlugin) Message(message msg.Message) bool {
 	tokens := strings.Fields(strings.ToLower(message.Body))
 	for i, token := range tokens {
 		if _, ok := p.Emoji[token]; ok {
-			emojied++
+			if token != "a" && token != "it" {
```
Comment (skiesel) [inline plugins/emojifyme/emojifyme.go]: Read these strings in from the global config stuff, and then stick them into a map and check them that way.

Comment (mccoyst) [reply plugins/emojifyme/emojifyme.go]: I thought you'd like hardcoding more… 😈

**Comment by mccoyst** [comment]
PTAL.

File: plugins/emojifyme/emojifyme.go [hunk 2]
```diff
@@ -114,3 +121,12 @@ func (p *EmojifyMePlugin) RegisterWeb() *string {
 }
 
 func (p *EmojifyMePlugin) ReplyMessage(message msg.Message, identifier string) bool { return false }
+
+func stringsContain(haystack []string, needle string) bool {
```
Comment (chrissexton) [inline plugins/emojifyme/emojifyme.go]: :authorized:

**Pull request merged**